release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1694,6 +1694,25 @@ In the following tables, features are marked with the following statuses:
 
 * When running a cluster with FIPS enabled, you might receive the following error when running an older version of the OpenShift CLI (`oc`) on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. As a workaround, use the `oc` binary provided with the {product-title} cluster. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*])
 
+* When you run CNF latency tests on an {product-title} cluster, the `oslat` test can sometimes return results greater than 20 microseconds. This results in an `oslat` test failure.
+(link:https://issues.redhat.com/browse/RHEL-9279[*RHEL-9279*])
+
+* When you use `preempt-rt` patches with the realtime kernel and you update the SMP affinity of a network interrupt, the corresponding IRQ thread does not immediately receive the update.
```

Comment:
Also this 4.15 known issue - @linuxqiao 
Again this is a direct copy of the 4.14 known issue note for https://issues.redhat.com/browse/RHELPLAN-138733 please